libs/mibao-ui/src/lib/image/image.tsx

```diff
@@ -11,13 +11,19 @@ import { useState, useMemo, useEffect, useCallback, ReactNode } from 'react'
 import FALLBACK_SRC from '../../../assets/images/fallback.svg'
 import { addParamsToUrl, disableImageContext, getImagePreviewUrl } from '../../utils'
 
+export type ImageSize = 'favicon' | 'xx-small' | 'x-small' | 'small' | 'medium' | 'large' | 'x-large' | 'xx-large' | 'xxx-large'
```

Comment:
why not keep consistent with the `size` of `chakra`, like `sm`, `md`, `lg`, `xl`